scripts/generic-ci-tests.sh

```diff
@@ -101,14 +101,12 @@ case "$TEST_SUITE" in
             "all")
                 paver test_system -s lms --extra_args="--with-flaky" --cov_args="-p"
                 ;;
-            "1")
-                paver test_system -s lms --extra_args="--attr='shard_1' --with-flaky" --cov_args="-p"
-                ;;
-            "2")
-                paver test_system -s lms --extra_args="--attr='shard_2' --with-flaky" --cov_args="-p"
+
+            [1-3])
```

Comment:
Bash doesn't distinguish, afaik. Looking at the test runs, this seems to work, except that my attr selector seems to be broken (no tests actually run)